tests/sentry/api/endpoints/test_organization_releases.py

```diff
@@ -911,6 +914,132 @@ def test_bad_repo_name(self):
         assert response.data == {'refs': [u'Invalid repository names: not_a_repo']}
 
 
+class OrganizationReleaseCommitRangesTest(SetRefsTestCase):
```

Comment:
Note that I only test for OrganizationReleases because ProjectReleases never had refs.